README.md

```diff
@@ -63,6 +119,12 @@ issues:
         - kubeapilinter
 ```
 
+#### Optional Configurations
+
+##### Enabling Specific Linters (Valid for Versions < 2.x)
+
+> **Note**: The following configuration is only supported when using the **module integration** via `.custom-gcl.yml` and is **not valid in plugin mode** (`.so` files with `golangci-lint` v2.x+).
```

Comment:
This isn't true, you can do exactly the same configuration between both versions

Reply:
I tried it, but it did not work out.
Now, with your message, I found: https://github.com/golangci/golangci-lint/blob/main/.custom-gcl.reference.yml

It might changed the format. I will try out after.

Reply:
I could not make work yet, more info: https://github.com/kubernetes-sigs/kubebuilder/pull/4941#discussion_r2217566613

Reply:
I was trying but I could not make work yet see: https://github.com/kubernetes-sigs/kubebuilder/pull/4941
(I will spend more effort on it later, but if you'd like, take a look. (Not sure what I am missing)